tools/c7n_kube/c7n_kube/actions/replicas.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018-2019 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from c7n_kube.actions.core import PatchAction
+from c7n.utils import type_schema
+log = logging.getLogger('custodian.k8s.replicas')
+
+
+class ReplicaAction(PatchAction):
+    """
+    Scale a resource
+
+    .. code-block:: yaml
+
+      policies:
+        - name: replica-resource
+          resource: k8s.deployment # k8s.{resource}
+          filters:
+            - 'metadata.name': 'name'
+          actions:
+            - type: replica
```

Comment:
how is this different then the more generically useful extant action for patch?

```
type: patch
options:
   replicas: 1
```
 
we don't want to pollute the action namespace with hundreds of attribute level manipulation actions, versus one generic one that can handle all of them.

Reply:
Not so different, but since I didn't found information about how to properly use the patch feature.

You're right on do not pollute the action namespace, but I think this is the most common use case. 
I'm ok if you want me to drop the PR and contribute with documentation about how to implement it.